stage0/run.go

```diff
@@ -211,6 +204,29 @@ func Run(dir string, debug bool) {
 	}
 }
 
+func makeUniqueContainer(pdir string) (*types.UUID, string, error) {
```

Comment:
Does "makeUniqueContainerDir" or "makeUniqueDir" make more sense, since this only creates a directory?